mindmeld/text_preparation/text_preparation_pipeline.py

```diff
@@ -82,6 +82,19 @@ def preprocess(self, text):
             preprocessed_text = preprocessor.process(preprocessed_text)
         return preprocessed_text
 
+    def custom_preprocessors_exist(self):
+        """ Checks if the current TextPreparationPipeline has preprocessors that is not
+        simply the NoOpPreprocessor or None.
+
+        Returns:
+            has_custom_preprocessors (bool): Whether atleast one custom preprocessor exists.
+        """
+        return (
+            self.preprocessors is not None
+            and len(self.preprocessors) >= 1
+            and not isinstance(self.preprocessors[0], NoOpPreprocessor)
```

Comment:
I think this should be `and not any([isinstance(elem, NoOpPreprocessor) for elem in self.preprocessors])`